setup.cfg

```diff
@@ -0,0 +1,76 @@
+[metadata]
+name = imagecraft
+version = attr: imagecraft.__version__
+description="Create Ubuntu bootable images."
+url = https://github.com/canonical/imagecraft
+project_urls =
+    Source = https://github.com/canonical/imagecraft.git
+    Issues = https://github.com/canonical/imagecraft/issues
+author = Canonical Ltd.
+author_email = snapcraft@lists.snapcraft.io
```

Comment:
Should we maybe put the ubuntu-devel mailing list here instead or something..?

Reply:
I thought about that too but I used the same email I found in other craft apps. I suppose we probably won't package and distribute the tool under another format than a snap, so I am not even sure this value will ever be used by a user to contact us. I see this as part of the "template" of a `setup.cfg` file to use when packaging a craft snap. 

@cmatsuoka what do you think about it?